opt-may-2016/Nipulpatel/com/assignment7/java/ToyotaDealer.java

```diff
@@ -0,0 +1,31 @@
+/***************************************************************
+ * @author Nipul								 date:6/28/2016 
+ * project:Assignment 7
+ * Scope : ToyotaDealer  which only can sell Toyota Cars.
+ * Scope change : none
+ * class : Sub class of car 
+ ***************************************************************/
+package com.assignment7.java;
+
+public class ToyotaDealer extends Car {

```

Comment:
but still toyotadealer is a stand alone class...it need not extend anything..all it will have is a sellCar(Toyota toyotaCar) method